app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUiRenderer.kt

```diff
@@ -35,6 +35,7 @@ class BloodSugarEntryUiRenderer(
 
   private fun showEditBloodSugarTitle(measurementType: BloodSugarMeasurementType) {
     with(ui) {
+      showRemoveButton()
```

Comment:
The name of this method is `showEditBloodSugarTitle`, but it also does showing of the remove button. Given that the showing of the remove button is not conditionally triggered based on the `measurementType`, it seems to me that this method call here is because of convenience and not because it actually belongs here.

What do you think about moving this into the `render()` method instead? We can just decide to show/hide this based on the `openAs` property.

Reply:
Moved the `showRemoveButton` inside the `setupUi` method